Sources/ProjectDescription/TestPlan.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public struct TestPlanList: Equatable, Codable {
```

Comment:
it's less explicit, but I guess it doesn't matter for most cases.
actually I'm not even sure it makes sense to support multiple plans at all. I can't imagine a case where this would make sense 🤔 .

Reply:
done